backend/api/Areas/Persons/Controllers/PersonController.cs

```diff
@@ -54,7 +73,7 @@ public PersonController(IPimsRepository pimsService, IMapper mapper)
         public IActionResult AddPerson([FromBody] Models.Person.PersonModel model, bool userOverride = false)
         {
             // Business rule - support country free-form value if country code is "Other". Ignore field otherwise.
-            var otherCountry = _pimsService.Lookup.GetCountries().FirstOrDefault(x => x.Code == Dal.Entities.CountryCodes.Other);
+            var otherCountry = _pimsRepository.Lookup.GetCountries().FirstOrDefault(x => x.Code == Dal.Entities.CountryCodes.Other);
```

Comment:
Don't have to change this now - but going forward we don't want controllers to access repositories directly.

Reply:
(The service would handle operations like this).

Reply:
I know but notice this is for Add person (which I didn't want to refactor here). Agree, data access should be moved/proxied to services